apps/sms/services/test/unit/moz_mobile_connections/moz_mobile_connections_client_test.js

```diff
@@ -0,0 +1,76 @@
+/*global bridge,
+         BroadcastChannel,
+         MozMobileConnectionsClient,
+         MocksHelper
+*/
+
+'use strict';
+
+require('/services/test/unit/mock_bridge.js');
+require('/views/shared/test/unit/mock_broadcast_channel.js');
+require('/services/js/moz_mobile_connections/moz_mobile_connections_client.js');
+
+var MocksHelperForAttachment = new MocksHelper([
+  'bridge',
+  'BroadcastChannel'
+]).init();
+
+suite('MozMobileConnectionsClient >', function() {
+  var clientStub;
+  var APP_INSTANCE_ID = 1;
```

Comment:
__optional nit:__ ```init``` expects ```string``` type :) Just for the clarity, but not important at all.